exonum-java-testing/pom.xml

```diff
@@ -43,6 +43,18 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-compiler-plugin</artifactId>
+        <configuration>
```

Comment:
Why can't this configuration be inherited from the parent POM?

Reply:
It can. But if we make configuration inheritance :
`mvn clean install `- will work with error-prone
`mvn clean install -pl :exonum-java-binding-common` - won't work with error-prone
Do we need it separately for submodules ?

Reply:
Got it , need to specify `combine.children="append"` for correct configuration group.